classes/class-p4-settings.php

```diff
@@ -0,0 +1,193 @@
+<?php
+
+if ( ! class_exists( 'P4_Settings' ) ) {
+	/**
+	 * Class P4_Settings
+	 */
+	class P4_Settings {
+
+		/**
+		 * Option key, and option page slug
+		 * @var string
+		 */
+		private $key = 'planet4_options';
+
+		/**
+		 * Array of metaboxes/fields
+		 * @var array
+		 */
+		protected $option_metabox = [];
+
+		/**
+		 * Options Page title
+		 * @var string
+		 */
+		protected $title = '';
+
+		/**
+		 * Options Page hook
+		 * @var string
+		 */
+		protected $options_page = '';
+
+		/**
+		 * Constructor
+		 */
+		public function __construct() {
+
+			// Set our title
+			$this->title   = __( 'Planet4', 'planet4-master-theme' );
+
+			$this->fields = apply_filters( 'planet4_options', [
+				[
+					'name'    => __( 'Select act page', 'planet4-master-theme' ),
+					'id'      => 'select_act_page',
```

Comment:
As noted in the blocks plugin, please change the name from "select_act_page" to "act_page"